modules/Components/src/server/routes/CodeTableServiceRoutes.coffee

```diff
@@ -37,31 +37,33 @@ exports.getAllCodeTableValues = (req, resp) ->
 		)
 
 exports.getCodeTableValues = (req, resp) ->
-	exports.getCodeTableValuesInternal req.params.type, req.params.kind, (result) ->
+	exports.getCodeTableValuesInternal req.params.type, req.params.kind, req.query, (result) ->
```

Comment:
This and the few lines below are the main changes on the acas side.  I added backwards compatability for the old syntax so out of date customer specific server functions would work the same.  This passes the query parameters directly to roo.